sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -482,16 +482,29 @@ Text Analytics for health is a containerized service that extracts and labels re
 The Analyze functionality allows to choose which of the supported Text Analytics features to execute in the same set of documents. Currently the supported features are: entity recognition, key phrase extraction, and Personally Identifiable Information (PII) Recognition. For more information see [How to: Use Text Analytics for analyze operation][analyze_operation_howto].
 
 ```C# Snippet:AnalyzeOperationBatchConvenience
-    string document = @"We went to Contoso Steakhouse located at midtown NYC last week for a dinner party, 
```

Comment:
I just noticed that the tittle of this section is `Run Analyze Operation Asynchronously`. could you modify it to use the async version of this code?

Reply:
@maririos I found the same thing in Sample_AnalyzeBatchActions.md so I changed that too